plugins/callback/selective.py

```diff
@@ -115,8 +115,8 @@ def _print_task(self, task_name=None):
             line_length = 120
             if self.last_skipped:
                 print()
-            msg = colorize("# {0} {1}".format(task_name,
-                                              '*' * (line_length - len(task_name))), 'bold')
+            line = "# {0} ".format(task_name)
+            msg = colorize("{0}{1}".format(line, '*' * (line_length - len(line))), 'bold')
```

Comment:
Wouldn't it be simpler to go

```suggestion
            msg = colorize("{0}{1}".format(line, '*' * (line_length - len(line) - 3)), 'bold')
```
?

Reply:
Yes, however if somebody ever comes along and changes the formatting and it uses a different number of characters they would need to manually update the magic number in that comparison. I figured this was better and also felt more inline with the style of the rest of the file (e.g. all of the `_print_host_or_item()` func)

Reply:
Fair enough